src/app/containers/RelatedTopics/index.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useContext } from 'react';
+import { TopicTag, TopicTags } from '@bbc/psammead-topic-tags';
+import { pathOr } from 'ramda';
+import SectionLabel from '@bbc/psammead-section-label';
+import styled from '@emotion/styled';
+import { arrayOf, shape, string, node } from 'prop-types';
+import { GelPageGrid, GridItemLarge } from '#components/Grid';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import { RequestContext } from '#app/contexts/RequestContext';
+import useClickTrackerHandler from '#hooks/useClickTrackerHandler';
+import useViewTracker from '#hooks/useViewTracker';
+import useToggle from '#hooks/useToggle';
+
+const eventTrackingData = {
+  componentName: 'topics',
+};
+
+const StyledTopicsWrapper = styled.aside`
+  padding-bottom: 2.5rem;
+`;
+
+const StyledSectionLabel = styled(SectionLabel)`
+  margin-top: 0;
+`;
+
+export const RelatedTopicsWrapper = ({ children }) => (
+  <GelPageGrid
+    enableGelGutters
+    columns={{
+      group0: 6,
+      group1: 6,
+      group2: 6,
+      group3: 6,
+      group4: 8,
+      group5: 20,
+    }}
+  >
+    <GridItemLarge>{children}</GridItemLarge>
+  </GelPageGrid>
```

Comment:
I've checked this out and ran and the related topics section seems to span 8 columns regardless of breakpoint e.g.
![image](https://user-images.githubusercontent.com/9645462/124301402-af9ef500-db57-11eb-8566-7ec80daaa15c.png)
Just not sure where these settings apply?

it looks sensible to be fair just not understanding..

Reply:
Yeah I had to investigate this since I didn't understand it either haha. So most of the other sections like related content etc use the `<CpsOnwardJourney>` component, which gets wrapped in a `<GelPageGrid>` and a `<GridItemLarge>`, which interestingly are both psammead-grid components, but `<GridItemLarge>` has the prop `item` set to `true`. This means that the topic tags section will be equivalently responsive to the other sections.

Reply:
Would you be able to take me through this in the swarm review later, I still don't really follow? Thanks 😅

Reply:
Haha yeah I'll give it a go 😂